star-ros2/src/star_bringup/README.md

```
@@ -0,0 +1,145 @@
+- `planner_frequency: 0.5` Hz frontier re-evaluation
+- `min_frontier_size: 0.75` m (ignore tiny openings)
+- `progress_timeout: 30.0` s (give up on unreachable frontiers)
+
+### `urdf/star.urdf.xacro`
+Robot URDF defining the sensor frame tree. Loaded by `robot_state_publisher`.
+
+---
+
+## TF Tree
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)